api/src/main/java/io/minio/MinioClient.java

```diff
@@ -927,6 +937,23 @@ private HttpResponse execute(Method method, String region, String bucketName, St
       this.traceStream.println(headers);
     }
 
+    SSLContext context = SSLContext.getInstance("TLSv1.2");
+
+    try {
+      context.init(null, null, null);
+    } catch (KeyManagementException e) {
```

Comment:
You can raise `KeyManagementException` itself than eating with `InternalException`